### DISCUSSION TITLE
New translation in Romanian

### DISCUSSION THREAD

**Discussion by baciucristian** [sircharlo/meeting-media-manager#1566] [Translations]
### What language do you speak?

I would like to help translate M³ into Romanian.

**Comment by mtdvlpr** [comment]
@baciucristian, thanks for reaching out! Please create an account (using GitHub) at [Crowdin](https://crowdin.com) and go to [this page](https://crowdin.com/project/meeting-media-manager/ro) to request access to our translation project.

**Comment by mtdvlpr** [comment]
Hi @baciucristian, if you want to see your translations in action before the next stable release, there's a beta version scheduled to automatically be released in 5 and half hours with the latest changes at that moment. To automatically receive that update, enable the `enable beta updates` option in the settings.

**Comment by baciucristian** [comment]
Translation done for the app.
